sigmastate/src/main/scala/sigmastate/serialization/ParameterSerializer.scala

```diff
@@ -0,0 +1,29 @@
+package sigmastate.serialization
+
+import sigmastate.Values.Parameter
+import sigmastate._
+import sigmastate.utils.{SigmaByteReader, SigmaByteWriter}
+
+import java.nio.charset.StandardCharsets
+
+object ParameterSerializer {
+  import sigmastate.Operations.ConstantPlaceholderInfo._
+
+  def serialize[T <: SType](parameter: Parameter, w: SigmaByteWriter): Unit = {
+    w.putUInt(parameter.name.length)
+    w.putBytes(parameter.name.getBytes(StandardCharsets.UTF_8))
```

Comment:
parameter.name.length is not always the number of bytes.

Reply:
yeah, thanks. I didn't know. Have updated the code to convert to bytes based on UTF_8 encoding.